README.md

```diff
@@ -104,41 +118,49 @@ val fetchThree: Fetch[(String, String, String)] = (fetchString(1) |@| fetchStrin
 When executing the above fetch, note how the three identities get batched and the data source is only queried once.
```

Comment:
Since identities are batched `fetchMany` is used to get them. The point is that you don't have to manually call `fetchMany`, i do it for you when there is many things of the same type to fetch.